scanners/helpers.js

```diff
@@ -7,16 +7,27 @@ const k8s = require("@kubernetes/client-node");
 const kc = new k8s.KubeConfig();
 kc.loadFromDefault();
 
-const k8sCRDApi = kc.makeApiClient(k8s.CustomObjectsApi);
-const k8sBatchApi = kc.makeApiClient(k8s.BatchV1Api);
-const k8sPodsApi = kc.makeApiClient(k8s.CoreV1Api);
+let k8sCRDApi, k8sBatchApi, k8sPodsApi;
 
+function getKubernetesAPIs() {
+  if (!k8sCRDApi) {
+    k8sCRDApi = kc.makeApiClient(k8s.CustomObjectsApi);
+  }
+  if (!k8sBatchApi) {
+    k8sBatchApi = kc.makeApiClient(k8s.BatchV1Api);
+  }
+  if (!k8sPodsApi) {
+    k8sPodsApi = kc.makeApiClient(k8s.CoreV1Api);
+  }
+
+  return { k8sCRDApi, k8sBatchApi, k8sPodsApi };
+}
 let namespace = "integration-tests";
 
 const sleep = (ms) => new Promise((resolve) => setTimeout(resolve, ms * 1000));
 
-async function deleteScan(name) {
-  await k8sCRDApi.deleteNamespacedCustomObject(
+async function deleteScan(name, CRDApi = getKubernetesAPIs().k8sCRDApi) {
```

Comment:
I think I'd personally change these arguments in all of these functions to get in the entire getKubernetesAPIs as one bundle.

So basically every function would get a argument called k8sApis which then has the CrdApi, BatchApi, PodsApi on it. (Like it's already returned by the `getKubernetesAPIs()` function) Passing these as seperate arguments seems more complicated to me